app/pages/_app.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import type { AppProps } from "next/app";
+import Theme from "../assets/global-styles/theme"
```

Comment:
In the mobile version, the bullet titles wrap strangely. Making it look like something is broken

![Screenshot 2023-02-20 at 1 36 22 PM](https://user-images.githubusercontent.com/18233774/220048527-77f770e7-7d32-44f5-8267-3493e957171b.png)